easybuild/main.py

```diff
@@ -601,7 +604,7 @@ def process_eb_args(eb_args, eb_go, cfg_settings, modtool, testing, init_session
     success_msg = "Build succeeded "
     if build_option('ignore_test_failure'):
         success_msg += "(with --ignore-test-failure) "
-    success_msg += "for %s out of %s" % (correct_builds_cnt, len(ordered_ecs))
+    success_msg += f"for {correct_builds_cnt} out of {len(ordered_ecs)} in {time2str(datetime.now() - start_time)}"
```

Comment:
Maybe we could use the same style as other time reporting eg for the steps of `... (took TIME)`?

Reply:
I'd be fine also having it on one line, maybe we could do something like

> Build succeeded for 5 out of 5 (total 46 mins 13 secs)

Also no strong opinion on it (could always be adjusted later)  but agree it would be nice to have the total runtime of the build as well.

Maybe for a separate PR, but related to this it might also be worth to get a total CPU/node time for parallel jobs